src/pegjs/floip.pegjs

```diff
@@ -20,20 +20,22 @@
     ];
   };
   ?> **/
-  const method = function(call, args, location) {
+  const method = function(call, args, chain, location) {
```

Comment:
Question: Does adding chain at parameter position 3, instead of parameter position 4 introduce any compatibility changes? I assume we need a minor version bump in order to make sure we don't mix versions? Or I could be wrong here.

Reply:
This will not result in any compatibility issues.
The function is only used in [one place](https://github.com/FLOIP/expression-parser/pull/11/files/421b69d7e04cea999cf67a43f1a21bba026b5d9f#diff-f6513a6fa7396efda80956eba5abe4dd0f8a1f6b831ee989377c5225e898caddR233) that has been updated to use the new signature.

The function _could_ be inlined, but I find having the AST element construction defined separately from the parsing to be useful.